src/rust/ide/view/graph-editor/src/component/breadcrumbs/project_name.rs

```diff
@@ -327,15 +335,22 @@ impl ProjectName {
             eval animations.color.value((value) model.set_color(*value));
             eval animations.position.value((value) model.set_position(*value));
 
+
              // === Pointer style ===
 
-             frp.output.source.pointer_style <+ model.view.events.mouse_over.map(|_|
+             editable <- all(&frp.output.edit_mode,&frp.ide_text_edit_mode).map(|(a,b)| *a || *b);
+             mouse_over_and_editable <- all(frp.output.is_hovered,editable).map(|(a,b)| *a && *b);
+             mouse_over_while_editing <- mouse_over_and_editable.gate(&mouse_over_and_editable);
```

Comment:
In such patterns, I like naming variables with `on_` prefix, like `on_mouse_over_and_editable` - this is what I use in my FRP networks. Just FYI, Im ok with leaving it as is, but if we want bigger consistency we can use one style here